### PR TITLE
Updated getUsersById for email

### PR DESCRIPTION
**Pull request by jalpa015**

### CODE REVIEW

File: test/integration/users.test.js [hunk 1]
```diff
@@ -241,8 +241,8 @@ describe("Users", function () {
           expect(res.body).to.be.a("object");
           expect(res.body.message).to.equal("User returned successfully!");
           expect(res.body.user).to.be.a("object");
-          expect(res.body.user).to.not.have.property("phone");
-          expect(res.body.user).to.not.have.property("email");
+          // expect(res.body.user).to.not.have.property("phone");
+          // expect(res.body.user).to.not.have.property("email");
```
Comment (ankushdharkar) [inline test/integration/users.test.js:245]: @jalpa015 Any reason why we want to keep dead code?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: test/integration/users.test.js (hunk 1 above)
Comment (jalpa015) [reply test/integration/users.test.js:245]: @ankushdharkar  I was unsure about making changes to tests file. So I left it commented. I will remove it.

Comment (jalpa015) [reply test/integration/users.test.js]: @ankushdharkar  Can I do something to fix the GithubSerive tests?

**Review by lakshayman — APPROVED** [review]
Looks good to me. 🙂
Thanks!

**Comment by lakshayman** [comment]
@jalpa015 we will be merging this by this weekend.

**Comment by jalpa015** [comment]
> @jalpa015 we will be merging this by this weekend.

Okay, thank you

**Comment by jalpa015** [comment]
Closes #578

**Pull request merged**